.github/workflows/ci.yml

```diff
@@ -25,19 +25,10 @@ jobs:
       run: cd test && make tests
     - name: python wheels
       run:
-        sudo apt-get -y install python3-pip python3-setuptools python3-venv
-        && python3 -m pip install git+https://github.com/henryiii/cibuildwheel@henryiii/chore/intreebuild
-        && make clean
-        && rm -rf inst liblo-0.31 && tar -xzf liblo-0.31.tar.gz
-        && python3 -m cibuildwheel --output-dir wheelhouse ./swig
-        && ls wheelhouse
-      env:
-        CIBW_BUILD: "cp3*"
-        CIBW_BEFORE_ALL: "yum install -y libtool automake autoconf zlib-devel swig"
-        CIBW_BEFORE_BUILD: "cd {package}; for package in $(pwd); do cd $package/../liblo-0.31 && ./configure --prefix=$package/../inst --disable-shared --enable-static --disable-tests --disable-tools --disable-debug --disable-examples CFLAGS=-fPIC && make clean && make install && cd $package/.. && env NOCONFIGURE=1 ./autogen.sh && ./configure PKG_CONFIG_PATH=$package/../inst/lib/pkgconfig --enable-static --disable-shared --disable-tests --disable-jni --disable-docs --disable-audio --disable-debug CFLAGS=-fPIC && make clean && make -C src && make -C swig mapper_wrap.c; done"
+        cd bindings/python && pip3 wheel . -w ./wheelhouse/
```

Comment:
Sorry I didn't get to looking at this before, but doesn't this change mean that the wheel no longer includes the library?  Not against removing SWIG and all its complexities (actually all for it), but the idea of the wheel was to include the libmapper binary so that it could be easily installed via pip.

Reply:
You're right – I started trying to use [auditwheel](https://github.com/pypa/auditwheel) and [delocate](https://pypi.org/project/delocate/) to automatically include libmapper and liblo but they weren't detecting the dependency – perhaps related to the ctypes bindings being a "pure python" wheel? I probably should have waited on the merge but got distracted & didn't want two slightly incompatible versions of the python bindings around...

Any chance you have more time to mess with the CI script and get the library included again?

Reply:
No problem, just noticed it.  I'm happy to take a shot, haven't had much time for hobby programming lately unfortunately.. i never got around to uploading to pypi!  but with your new version it's probably for the best, would have been premature.  now it's a good time, so I'll try to find some time for it.